core/frontend/src/components/app/ReportMenu.vue

```diff
@@ -52,7 +52,7 @@
             >
               $si-discourse
             </v-icon>
-            With Blue Robotics discuss
+            On Blue Robotics discuss
```

Comment:
```suggestion
            On Blue Robotics forum
```
Most people likely don't know that the forum happens to be using Discourse, and "discuss" is a verb (not a noun/place)

Reply:
Check now